exchanges/bitstamp/bitstamp_test.go

```diff
@@ -354,15 +353,19 @@ func TestGetOpenOrders(t *testing.T) {
 		sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
 	}
 	o, err := b.GetOpenOrders(context.Background(), "btcusd")
-	assert.NoError(t, err, "GetOpenOrders should not error")
+	require.NoError(t, err, "GetOpenOrders must not error")
 	if mockTests {
 		assert.NotEmpty(t, o, "Orders should not be empty")
 		for _, res := range o {
-			assert.Equal(t, "2022-01-31 14:43:15", res.DateTime, "order date should match")
-			assert.Equal(t, int64(1234123412341234), res.ID, "order ID should match")
-			assert.Equal(t, 0.50000000, res.Amount, "amount should match")
-			assert.Equal(t, 100.00, res.Price, "price should match")
-			assert.Equal(t, 0, res.Type, "type should match")
+			assert.Equal(t, "2022-01-31 14:43:15", res.DateTime, "DateTime should match")
+			assert.Equal(t, int64(1234123412341234), res.ID, "ID should match")
+			assert.Equal(t, 0.50000000, res.Amount, "Amount should match")
+			assert.Equal(t, 100.00, res.Price, "Price should match")
+			assert.Equal(t, 0, res.Type, "Type should match")
+			assert.Equal(t, 0.50000000, res.AmountAtCreate, "AmountAtCreate should match")
+			assert.Equal(t, 110.00, res.LimitPrice, "LimitPrice should match")
+			assert.Equal(t, "1234123412341234", res.ClientOrderID, "ClientOrderID should match")
+			assert.Equal(t, "BTC/USD", res.Market, "Market should match")
```

Comment:
One comment was regarding the Currency is not set for []Order type. Bitstamp open orders for markets do not include currency in the response so I have added "Market" which was missing